### ISSUE TITLE
use OEP API

### ISSUE THREAD

**Issue by nesnoj** [openego/eDisGo#83]
For access to OEP data, the [API](http://oep-data-interface.readthedocs.io/en/latest/api/how_to.html) should be used.

**Comment by gplssm** [comment]
I remember you mentioned some issues with the OEP-API and certain data types... How about that.

Further from that, shall we wait 'til the [SQLAlchemy dialect](https://github.com/openego/oeplatform/issues/261) is available or do we change to use the way using requests library?

**Comment by nesnoj** [comment]
Thanks for pulling me back to reality..
You're right, the API does not support the postGIS stuff (and I am not sure what else is missing).
In eDisGo our queries are quite simple - the only geo operation we use is ST_Transform. We could replace it by using pyproj..
My preferred solution is the SQL dialect but as far as I know, it won't be developed in open_eGo. So we cannot be sure that it's gonna be implemented (depending on future projects..).

Ok, let's put this on on hold..

**Comment by nesnoj** [comment]
SQL dialect will be available soon, see https://github.com/openego/oeplatform/issues/261#issuecomment-345749623

**Comment by gplssm** [comment]
Not connected to a release anymore. We do this right when the dialect is available.

**Comment by nesnoj** [comment]
As the dialect is available now, any updates here yet?

**Comment by gplssm** [comment]
I'm using it since appr. 2 weeks continously. Works for me. The required changes to eDisGo were done. Now, we should describe it somehow in the docs how to use it, because it is not self-explaining. Preferably, we decribe is in ego.io in order to do it only once.

**Comment by nesnoj** [comment]
Like you probably know, there's some (general) stuff documented at the RLI Jupyter Server (notebook OEP-Dialect.ipynb)

**Comment by nesnoj** [comment]
Related: https://github.com/openego/ego.io/issues/57
